subprojects/gradle/cd/src/gradleTest/kotlin/com/avito/ci/BaseCiStepsPluginTest.kt

```diff
@@ -0,0 +1,147 @@
+package com.avito.ci
+
+import com.avito.android.plugin.artifactory.artifactoryPasswordParameterName
+import com.avito.android.plugin.artifactory.artifactoryUserParameterName
+import com.avito.instrumentation.instrumentationPluginId
+import com.avito.test.gradle.TestProjectGenerator
+import com.avito.test.gradle.dependencies.GradleDependency
+import com.avito.test.gradle.module.AndroidAppModule
+import com.avito.test.gradle.module.AndroidLibModule
+import com.avito.test.gradle.plugin.plugins
+import java.io.File
+
+internal abstract class BaseCiStepsPluginTest {
```

Comment:
Extract because CiStepsTests has too many test methods.
So I decided to split it